includes/Core/CLI/Authentication_CLI_Command.php

```diff
@@ -2,37 +2,42 @@
 /**
  * Site Kit Authentication CLI Commands
  *
- * @package   Google\Site_Kit
- * @copyright 2019 Google LLC
+ * @package   Google\Site_Kit\Core\CLI
+ * @copyright 2020 Google LLC
  * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  * @link      https://sitekit.withgoogle.com
  */
 
-namespace Google\Site_Kit\CLI;
+namespace Google\Site_Kit\Core\CLI;
 
-use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Storage\Options;
 use Google\Site_Kit\Core\Storage\User_Options;
+use Google\Site_Kit\Core\Storage\Transients;
 use Google\Site_Kit\Core\Authentication\Authentication;
 use WP_CLI;
-use WP_CLI_Command;
-
-if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly.
-}
-
-WP_CLI::add_command( 'google-site-kit auth', __NAMESPACE__ . '\Authentication_CLI_Command' );
 
 /**
- * Site Kit Authentication CLI Command
+ * Manages Site Kit user authentication for Google APIs.
+ * 
+ * @since n.e.x.t
+ * @access private
+ * @ignore
  */
-class Authentication_CLI_Command extends WP_CLI_Command {
+class Authentication_CLI_Command extends CLI_Command {
 
 	/**
 	 * Logout and revoke refresh token
```

Comment:
This should use a third-person verb, and a period should be added. I wonder whether we should generally change this description though to match our terminology: "Disconnects a user from Site Kit, removing their relevant user options and revoking their token."

Also, how about we rename the method (and command) to `disconnect`? We could specify a command alias annotation so that `revoke` still works for back-compat, but that name is slightly off.

Reply:
Yep, agree. Updated the description and changed the command name to be `disconnect` (plus added `revoke` as an alias).